backend/Services/LocationService.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using Microsoft.AspNetCore.Cryptography.KeyDerivation;
+using WhaleSpotting.Repositories;
+
+namespace WhaleSpotting.Services
+{
+    public interface ILocationService
+    {
+    }
+    public class LocationService : ILocationService
+    {
+        private readonly ILocationRepo _locations;
+
+        public LocationService(ILocationRepo locations)
+        {
+            _locations = locations;
+        }
+    }
+};
```

Comment:
I don't think semicolons here are normal!
```suggestion
}
```

Reply:
It's just in the authService file we copied _shrug_